src/Models/LoginModel.php

```diff
@@ -40,7 +40,7 @@ class LoginModel extends BaseModel
         'identifier' => 'permit_empty|string',
         'user_agent' => 'permit_empty|string',
         'user_id'    => 'permit_empty|integer',
-        'date'       => 'required|valid_date',
+        'date'       => 'required',
```

Comment:
Do we need more validation rule?

Reply:
The value of **date** is not taken from the user. Therefore, I do not see the need `valid_date`.